app/inbox/models.py

```diff
@@ -1,30 +1,23 @@
 from django.db import models
+from economy.models import SuperModel
 from django.utils.translation import gettext_lazy as _
 from django.contrib.auth import get_user_model
-from django.forms.models import model_to_dict
 
 
-class Notification(models.Model):
-    created_on = models.DateTimeField(auto_now_add=True, editable=False)
-    modified_on = models.DateTimeField(auto_now=True)
-    CTA_URL = models.URLField(max_length=255, null=True)
-    CTA_Text = models.CharField(max_length=255)
-    message_html = models.CharField(max_length=255, null=True, help_text=_("Html message"))
+class Notification(SuperModel):
+
+    cta_url = models.URLField(max_length=255, blank=True)
+    cta_text = models.CharField(max_length=255)
+    message_html = models.CharField(max_length=255, blank=True, help_text=_("Html message"))
     is_read = models.BooleanField(default=False)
-    to_user_id = models.ForeignKey(
-            get_user_model(),
-            on_delete=models.CASCADE,
-            related_name='to_user'
+    to_user = models.ForeignKey(get_user_model(),
+                on_delete=models.CASCADE,
+                related_name='received_notification'
```

Comment:
Can we change this to received_notifications vs notification since it relates to n number of Notification objects? Also, as stickler states, the indentation needs fixed here.